origin-js/src/contractInterface/marketplace/v00_adapter.js

```diff
@@ -250,7 +269,9 @@ class V00_MarkeplaceAdapter {
       if (event.event === 'ListingCreated') {
         ipfsHash = event.returnValues.ipfsHash
       } else if (event.event === 'ListingUpdated') {
-        ipfsHash = event.returnValues.ipfsHash
+        if (!offerBlockNumber || offerBlockNumber < event.blockNumber) {
+          ipfsHash = event.returnValues.ipfsHash
+        }
```

Comment:
@wanderingstan - this is the code that enables us to show the listing data as it was when the listing was created for "my purchases" and "my sales"

Reply:
Actually, this may not be necessary - something is working differently than I expected... still looking into it... I'm trying to grok this code - I don't think I know what it's doing actually:
https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/components/my-sales.js#L36-L61

Reply:
lol - okay - yes, this code is needed and works now - temporarily thought I was losing my marbles 😅